src/controllers/verifyCommand.ts

```diff
@@ -1,8 +1,12 @@
+import { VERIFICATION_SITE_URL } from "../constants/urls";
 import { env } from "../typeDefinitions/default.types";
 import { discordTextResponse } from "../utils/discordResponse";
+import { generateUniqueToken } from "../utils/generateUniqueToken";
 import { sendDiscordDm } from "../utils/sendDiscordDm";
 
 export async function verifyCommand(userId: number, env: env) {
-  await sendDiscordDm(userId, env);
+  const token = await generateUniqueToken();
+  const url = `${VERIFICATION_SITE_URL}/?token=${token}`;
+  await sendDiscordDm(userId, env, url);
```

Comment:
The third parameter for `sendDiscordDm` currently is message. Can you please re verify.

Reply:
yes